runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -233,8 +237,9 @@ public V get(@ConflictParameter Object key) {
     Collection<Entry<K, V>> getByIndex(@Nonnull Index.Name indexName, I indexKey) {
         String secondaryIndex = indexName.get();
         Map<Object, Map<K, V>> secondaryMap;
-        if (secondaryIndexes.containsKey(secondaryIndex) &&
-                ((secondaryMap = secondaryIndexes.get(secondaryIndex)) != null)) {
+        if ((secondaryIndexes.containsKey(secondaryIndex) &&
```

Comment:
if using an alias, "secondaryIndex" will be the alias, correct?

Reply:
yes, "secondaryIndex" is either a fully qualified name or an alias, whatever was provided in the call (both are valid)